PokemonAPI/PokemonAPI.BusinessLayer/Validator/PokemonValidator.cs

```diff
@@ -0,0 +1,107 @@
+﻿using PokemonAPI.BusinessLayer.Exceptions;
+using PokemonAPI.BusinessLayer.Interfaces;
+using PokemonAPI.DomainLayer.Entities;
+using PokemonAPI.PersistenceLayer.DTOModels;
+using System;
+using System.Collections.Generic;
+
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace PokemonAPI.BusinessLayer.Validator
+{
+    class PokemonValidator : IPokemonValidator
+    {
+        Dictionary<string, List<string>> errorMap = new Dictionary<string, List<string>>();
```

Comment:
specify access modifier better. By default this is private but it's more concise if it's specified

Reply:
do we really need this as class state?